doc/source/reference/maskedarray.baseclass.rst

```diff
@@ -20,10 +23,10 @@ defines several constants.
       >>> x[1] is ma.masked
       True
       >>> x[-1] = ma.masked
-      >>> x
-      masked_array(data = [1 -- --],
-                   mask = [False  True  True],
-             fill_value = 999999)
+      >>> x # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Do we need this? I'd rather we get the whitespace correct

Reply:
This is not needed. From reading `tools/refguide_check.py`, it seems that this flag is always on anyway.